<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Backup Engine</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Source Paths (separate with ;):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="sourceLineEdit">
      <property name="text">
       <string>C:\\source of backup\\</string>
      </property>
      <property name="placeholderText">
       <string>C:\source1; C:\source2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Destination Paths (separate with ;):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="destinationLineEdit">
      <property name="text">
       <string>C:\\backup\\</string>
      </property>
      <property name="placeholderText">
       <string>C:\backup1; C:\backup2</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start Backup</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Stop Backup</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Ready</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_StartBackup_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_StopBackup_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_StartBackup_clicked()</slot>
  <slot>on_StopBackup_clicked()</slot>
 </slots>
</ui>
